apps/simple-trading-app/src/app/components/simple-market-list/simple-market-list.tsx

```diff
@@ -7,8 +7,10 @@ import SimpleMarketPercentChange from './simple-market-percent-change';
 import SimpleMarketExpires from './simple-market-expires';
 import DataProvider from './data-provider';
 import { MARKET_STATUS } from './constants';
+import { useNavigate } from 'react-router-dom';
```

Comment:
non-blocking: Let's keep imports of external libs at the top of import section, here as well like above